src/ui-kit/SliderYears/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import { getEndYear, getStartYear } from 'redux/selectors'
+import { setCurrentYearsDate, setCurrentYearsAfter } from 'redux/actions'
+
+import { number } from 'prop-types'
+
+import { Slider } from 'antd'
+
+import './sliderYears.scss'
+
+
+const SliderYears = ({ setCurrentYearsDate, setCurrentYearsAfter, startYear, endYear }) => {
+
+  const currentDate = new Date()
+  const currentYear = currentDate.getFullYear()
+
+  const setYearsHandler = (value) => {
+      setCurrentYearsDate(value)
+  }
+
+  const setYearsAfterChange = value => {
+    setCurrentYearsAfter(value)
+  }
+
+  return (
+    <div>
+      <Slider
+        className='ui-kit__sliderYears'
+        range
+        delay
+        step={1}
+        min={1920}
+        max={currentYear}
+        defaultValue={[startYear, endYear]}
+        onChange={setYearsHandler}
+        onAfterChange={setYearsAfterChange}
+      />
+    </div>
+  )
+}
+
+
+const mapStateToProps = state => ({
+  startYear: getStartYear(state),
+  endYear: getEndYear(state),
+})
+
+const mapDispatchToProps = {
+  setCurrentYearsDate,
+  setCurrentYearsAfter,
+}
```

Comment:
extract **mapStateToProps** and **mapDispatchToProps ** away from this component, you should pass them through props

Reply:
all the components from **ui-kit** should not have **mapStateToProps** and **mapDispatchToProps**, could yu check and change that for all the components please @UtopiaEH  ??